test/end-to-end.js

```diff
@@ -1185,3 +1204,9 @@ function readBuiltinPhasesFromSanbox() {
   app.lazyrouter(); // initialize request handling phases
   return app._requestHandlingPhases;
 }
+
+function getSandBoxApp() {
+  var app = require(SANDBOX);
+  app.use(app.get('restApiRoot'), app.loopback.rest());
```

Comment:
Yes, he is using that line as a hack to load it on windows. But yes, I can also agree with skipping the entire test for now since boot isn't loading it. I'm ok with:

1. load only for windows tests
2. skip the test altogether (as you suggested)